packages/fxa-admin-server/package.json

```diff
@@ -61,11 +61,12 @@
     "mocha": "^7.1.0",
     "nodemon": "^2.0.2",
     "npm-run-all": "^4.1.5",
+    "pm2": "^4.2.3",
     "prettier": "^1.19.1",
     "proxyquire": "^2.1.3",
     "sinon": "^9.0.1",
     "supertest": "^4.0.2",
-    "ts-node-dev": "^1.0.0-pre.44",
+    "ts-node": "^8.8.2",
```

Comment:
A quick note on why I was using `ts-node-dev` here, from the webpage:

> Tweaked version of node-dev that uses ts-node under the hood.

> It restarts target node process when any of required files changes (as standard node-dev) but shares Typescript compilation process between restarts. This significantly increases speed of restarting comparing to node-dev -r ts-node/register ..., nodemon -x ts-node ... variations because there is no need to instantiate ts-node compilation each time.

This helps **quite a bit** when working on the code for quick restarts.